DepartmentController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+$__department = __DIR__ . '/Department.php';
+require_once $__department;
+
+$__employee = __DIR__ . '/Employee.php';
+require_once $__employee;
+
+class DepartmentController
+{
+
+    public function createWithEmployees(int $employee_count): Department
```

Comment:
Имена переменных должны быть в camelCase:
```suggestion
    public function createWithEmployees(int $employeeCount): Department
```